focal/debian/control

```diff
@@ -186,16 +203,52 @@ Description: Ignition Physics classes and functions for robot apps - TPE library
   .
   TPE library component shared library
 
+Package: libignition-physics5-bullet-dev
+Architecture: any
+Section: libdevel
+Depends: libignition-physics5-core-dev (= ${binary:Version}),
+         libignition-physics5-sdf-dev (= ${binary:Version}),
+         libignition-physics5-mesh-dev (= ${binary:Version}),
+         libignition-cmake2-dev (>= 2.1.0),
+         libignition-common4-graphics-dev,
+         libignition-math6-dev,
+         libignition-math6-eigen3-dev,
+         libignition-plugin-dev,
+         libbullet-dev,
+         libsdformat11-dev,
+         libignition-physics5-bullet (= ${binary:Version}),
+         ${misc:Depends}
+Multi-Arch: same
+Description: Ignition Physics classes and functions for robot apps - Development files
+  Ignition Physics is a component in the ignition framework, a set of libraries
+  designed to rapidly develop robot applications.
+  .
+  Bullet component, development files
+
+Package: libignition-physics5-bullet
+Architecture: any
+Section: libs
+Pre-Depends: ${misc:Pre-Depends}
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Multi-Arch: same
+Description: Ignition Physics classes and functions for robot apps - Bullet engine
+  Ignition Physics is a component in the ignition framework, a set of libraries
+  designed to rapidly develop robot applications.
+  .
+  Bullet component shared library
+
 Package: libignition-physics5-dev
 Architecture: any
 Section: libdevel
-Depends: libignition-physics5-core-dev,
-         libignition-physics5-dartsim-dev,
-         libignition-physics5-mesh-dev,
-         libignition-physics5-sdf-dev,
-         libignition-physics5-tpe-dev,
-         libignition-physics5-tpelib-dev,
+Depends: libignition-physics5-core-dev (= ${binary:Version}),
+         libignition-physics5-dartsim-dev (= ${binary:Version}),
+         libignition-physics5-heightmap-dev (= ${binary:Version}),
+         libignition-physics5-mesh-dev (= ${binary:Version}),
+         libignition-physics5-sdf-dev (= ${binary:Version}),
+         libignition-physics5-tpe-dev (= ${binary:Version}),
+         libignition-physics5-tpelib-dev (= ${binary:Version}),
          ${misc:Depends}
+# Bullet not added on purpose to avoid adding a new dependency to a stable release
```

Comment:
We can add it here, since it's not stable yet, what do you think?

Reply:
ok sure